src/lib.rs

```diff
@@ -1,8 +1,53 @@
 // Copyright 2024 the Velato Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
+//! Render a Lottie animation to a Vello [`Scene`](https://docs.rs/vello/*/vello/struct.Scene.html).
+//!
+//! However, this is also intended to be the preferred integration between Vello and [Lottie](https://lottie.github.io/lottie-spec/), so [consider
+//! contributing](https://github.com/linebender/velato) if you need a feature which is missing.
+//!
+//! This crate also re-exports [`vello`], to make handling dependency versions easier
+//!
+//! ## Usage
+//!
+//! ```no_run
+//! # use std::str::FromStr;
+//! # use velato::vello::kurbo::Affine;
```

Comment:
I'd probably unhide this line, because it's non-obvious where `Affine` comes from.

Additionally, I'd have a line like:
```rust
use velato::vello;
```
to show where the re-export occured

Reply:
fixed